pycaret/containers/models/classification.py

```diff
@@ -1147,8 +1147,15 @@ def __init__(self, experiment):
             "n_jobs": experiment.n_jobs_param,
             "verbosity": 0,
             "booster": "gbtree",
-            "tree_method": "gpu_hist" if experiment.gpu_param else "auto",
         }
+
+        # If XGBoost >= 2 change and add new parameters
+        if version.parse(xgboost.__version__) >= version.parse("2.0.0"):
+            args["tree_method"] = "hist"
```

Comment:
Looking at the docs [here](https://xgboost.readthedocs.io/en/stable/treemethod.html), I would think leaving `tree_method` to its default value is probably the safest option.

Reply:
I'm a little confused, I saw it here: https://github.com/dmlc/xgboost/blob/master/NEWS.md#hist-is-now-the-default-tree-method

Reply:
Yup, the `hist` is now the default. All tree methods are still available.

Reply:
Let's remove the `args["tree_method"]` line entirely then. Not specifying it uses the default value.